src/rgw/rgw_admin.cc

```diff
@@ -350,6 +350,7 @@ void usage()
   cout << "                             set list of zones to sync from\n";
   cout << "   --sync-from-rm=[zone-name][,...]\n";
   cout << "                             remove zones from list of zones to sync from\n";
+  cout << "   --bucket-index-max-shards override a zone/zonegroup's default bucket index shard count\n";
```

Comment:
does this need to indicate that there is a number included in the command-line-argument?

Reply:
i'm torn on this. some options do specify that they require a certain argument, and others don't. in this case, it would feel overly verbose and need a line break. and if there is confusion, the error message is pretty clear:
```
$ radosgw-admin zone modify --bucket-index-max-shards
Option --bucket-index-max-shards requires an argument.
```